backend/tilavarauspalvelu/models/reservation_unit_image/model.py

```diff
@@ -46,7 +46,7 @@ class Meta:
         base_manager_name = "objects"
         verbose_name = _("reservation unit image")
         verbose_name_plural = _("reservation unit images")
-        ordering = ["pk"]
+        ordering = ["image_type"]
```

Comment:
Getting the correct order here is accidental ("M" in "main" is before "O" in "other" alphabetically), but that is likely fine. More robust way would be to order using a case-expression (see `BANNER_LEVEL_SORT_ORDER`), BUT I think this is fine as is.

Reply:
I initially thought of that as well, but as we will only have two `image_type`s this works just as well 😄